app/src/assets/css/modules/_tools.scss

```diff
@@ -0,0 +1,229 @@
+h1 {
```

Comment:
Similar comment here to [this one ](https://github.com/Duke-MatSci/materialsmine/pull/148#discussion_r834824854): if there's a way that we can wrap these styles in a class (e.g., `.toolSets {}` or something like that) it may help prevent the styling from affecting other classes unexpectedly. Would also potentially mean adding another class to the outermost divs in all of the html files of the tools though so possibly this is TD issue

Reply:
I wrapped them the same way as in #148, and in both cases I added a few classes to the outermost divs. I agree that simplifying (and maybe even completely rewriting) the styling is a TD issue, but that will be a large story ticket.